### ISSUE TITLE
Refresh Help Center

### ISSUE THREAD

**Issue by samajammin** [SuperblocksHQ/ethereum-studio#255]
## Summary

https://help.superblocks.com/en/collections/1865071-ethereum-studio

Audit each article to ensure descriptions & images are up to date.

## Motivation

UX - some of these articles are clearly outdated.

## Describe alternatives you've considered
None

## Additional context
None

**Comment by ChrisChinchilla** [comment]
I actually feel this has been a bit of a disconnect for some time anyway, there may even be a better way to handle our onboarding/support/FAQs

**Comment by samajammin** [comment]
I agree @ChrisChinchilla. It might be best to rethink how we onboard & support users holistically. For instance, #242 may largely reduce the need for a help center.

Another area we're considering: we will soon be launching a public Ethereum.org Discord server. We could create a #studio channel where we send people for realtime support (like the Documentation Gitter currently does, though we've all discussed that's probably not the most relevant forum for that).

Another option is to consolidate attention around Discourse. @javier-tarazaga already announced in their Superblocks Discord server that they'll be moving more activity to [their Discourse platform](https://discuss.superblocks.com/).
